contracts/DAORegistry.sol

```diff
@@ -1,19 +1,16 @@
 pragma solidity ^0.5.4;
 
-import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "@openzeppelin/upgrades/contracts/Initializable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol";
 
-contract DAORegistry is Ownable {
+contract DAORegistry is Initializable, Ownable {
 
     event Propose(address indexed _avatar);
     event Register(address indexed _avatar, string _name);
     event UnRegister(address indexed _avatar);
 
     mapping(string=>bool) private registry;
 
```

Comment:
how about adding : 

```
function initialize(address _owner)
    public
    initializer {
        Ownable.initialize(_owner);
    }
```

and add a test to check ownership

Reply:
done